internal/step/plugin/provider.go

```diff
@@ -85,6 +93,8 @@ const (
 	StageIDOutput StageID = "outputs"
 	// StageIDCrashed is a stage that indicates that the plugin has quit unexpectedly.
 	StageIDCrashed StageID = "crashed"
+	// StageIDStarting is a stage that indicates that the plugin execution has begun.
+	StageIDStarting StageID = "starting"
 )
 
 var deployingLifecycleStage = step.LifecycleStage{
```

Comment:
shouldn't the NextStages for the StageIDDeploy be StageIDStarting?

Reply:
I think so, good catch!

Reply:
It looks like maybe this was resolved when main was merged in?